src/probnum/filtsmooth/gaussfiltsmooth/kalman.py

```diff
@@ -46,9 +47,62 @@ def iterated_filtsmooth(
         Parameters
         ----------
         regression_problem
+        init_posterior
+            Initial posterior to linearize at. Defaults to computing a (non-iterated)
+            smoothing posterior, which amounts to linearizing at the prediction
+            random variable.
```

Comment:
super minor comment: I find this a bit hard to parse. But that might just be me reading this at the end of my workday... ;)

Reply:
Definitely feel free to ignore htis!